### PR TITLE
docs: usage section

### PR DESCRIPTION
**Pull request by JeremyTubongbanua**
**- What I did**
Add usage instructions for `sshnpd` and `sshnp`

**- How I did it**
Edit `README.md` (see changes)

**- How to verify it**
**Daemon instructions (sshnpd)**
image

**Client instructions (sshnp)**
image

### CODE REVIEW

**Comment by JeremyTubongbanua** [comment]
@L1AMcB 

If you could be my 2nd pair of eyes, that would be great!

SSHNPD
image



SSHNP
image

**Review by L1AMcB — CHANGES_REQUESTED** [review]
I believe the description section of SSHNPD that says  for -m, -manager (mandatory), "Managers atSign" should be changed to "Manager's atSign" since the noun is singular. 

Other than that looks great to me!

**Review by L1AMcB — CHANGES_REQUESTED** [review]
Only comment is that some of the descriptions start with a lowercase and some are uppercase in the sshnpd section while all of the descriptions in the sshnp section start with an uppercase. 

Other than that on line 115 the description for local port says "ssh port to listen on, on your local machine", is "listen on" a destination or is having "on" twice a typo?

Otherwise looks good!

File: README.md [hunk 1]
```diff
@@ -72,6 +72,53 @@ to see the logs (-v).
...
+| --[no-]username     | -u           | send username to the manager to allow sshnp to display username in command line |
+| --[no-]verbose      | -v           | More logging                                                                    |
+
+### sshnp (client)
+Run the binary file or the dart file:
+```
+./sshnp <args|flags>
+```
+```
+dart bin/sshnp.dart <args|flags>
```
Comment (cpswan) [inline README.md]: use `dart run` here as above

File: README.md [hunk 2]
```diff
@@ -72,6 +72,53 @@ to see the logs (-v).
 
 Thoughts/bugs/contributions via PR all very welcome!
 
+## Usage
+
+### sshnpd (daemon)
+Run the daemon binary file or the dart file:
+```
```
Comment (cpswan) [inline README.md]: code fences might look better with language specified e.g. `sh` or `bash`

**Review by cpswan — CHANGES_REQUESTED** [review]

**Review by cpswan — APPROVED** [review]

**Comment by JeremyTubongbanua** [comment]
> Only comment is that some of the descriptions start with a lowercase and some are uppercase in the sshnpd section while all of the descriptions in the sshnp section start with an uppercase.
> 
> Other than that on line 115 the description for local port says "ssh port to listen on, on your local machine", is "listen on" a destination or is having "on" twice a typo?
> 
> Otherwise looks good!

Hmm yeah the wording is a bit weird, but I think it's still right.
It's saying that the local port you provide will be the reverse ssh port to listen on, and the port must be on your local machine

**Pull request merged**